src/pages/Home.js

```diff
@@ -50,17 +69,82 @@ function Header() {
           Welcome{firstName ? `, ${firstName}` : ""}!
         </Typography>
 
-        <CurrentCarePlan />
+        <Stack>
+          <Typography variant="eyebrow">Current Plan</Typography>
+          {CarePlans}
+        </Stack>
       </Container>
     </Box>
   );
 }
 
+const HomeContent = ({ carePlan }) => {
+  const tabs = [
+    {
+      title: 'Tasks',
+      content: <TaskCards carePlan={carePlan} />,
+    },
+    {
+      title: 'Goals',
+      content: <GoalCards carePlan={carePlan} />,
+    },
+    {
+      title: 'About',
+      content: <AboutCard carePlan={carePlan} />,
+    },
+  ];
+
+  const [tab, setTab] = useState(tabs[0]);
+
+  if (!carePlan) {
+    return null;
+  }
+
+  return (
+    <ControlledTabs
+      tabs={tabs}
+      tab={tab}
+      handleTabChange={(_, tabTitle) => {
+        const tab = tabs.find(tab => tab.title === tabTitle);
+        setTab(tab);
+      }}
+    />
+  );
+}
+
 export default function Home() {
+  const { carePlans, isLoading, isError } = useCarePlans();
+  const { care_plan_id } = useParams();
+  
+  if (isLoading) {
+    return (
+      <Skeleton variant="rectangular" animation="wave" height={280} />
+    );
+  }
+
+  if (isError) {
+    return (
+      <div>Woops something went wrong...</div>
+    );
+  }
+
+  let currentCarePlan;
+  const openCarePlans = carePlans.filter(carePlan => carePlan.status === "active");
+  const completedCarePlans = carePlans.filter(carePlan => carePlan.status === "completed");
+  const allCarePlans = [...openCarePlans, ...completedCarePlans]
+
+  if (care_plan_id) {
+    currentCarePlan = allCarePlans.find(carePlan => carePlan.id === care_plan_id);
+  } else if (openCarePlans.length > 0) {
+    currentCarePlan = openCarePlans[0];
+  } else if (completedCarePlans.length > 0) {
+    currentCarePlan = completedCarePlans[0];
+  }
+
   return (
     <>
-      <Header />
-      <TabsWithContent />
+      <Header carePlan={currentCarePlan} carePlans={allCarePlans} />
```

Comment:
if you dont need carePlans param:
```suggestion
      <Header carePlan={currentCarePlan} />
```